src/main/java/io/vinyldns/java/model/membership/Group.java

```diff
@@ -0,0 +1,93 @@
+package io.vinyldns.java.model.membership;
+
+import java.util.Set;
+import org.joda.time.DateTime;
+
+public class Group {
+    private final String name, email, id;
+    private final String description;
+    private final DateTime created;
+    private final GroupStatus status;
+    private final Set<String> memberIds, adminUserIds;
+
+    public Group(String name, String email, String description, String id, DateTime created, GroupStatus status,
+                 Set<String> memberIds, Set<String> adminUserIds) {
+        this.name = name;
+        this.email = email;
+        this.description = description;
+        this.id = id;
+        this.created = created;
+        this.status = status;
+        this.memberIds = memberIds;
+        this.adminUserIds = adminUserIds;
+    }
+
+    public Group(String name, String email, String id, DateTime created, GroupStatus status,
```

Comment:
not good form to take a parameter out of the middle of the constructor (seems like you removed description here), especially as it is backfilled with another `String`.

Leave "optional" elements to the end of the constructor, then set them to a valid default value in the constructor itself.

Often times it is good form to have one constructor call another providing the defaults, as opposed to the `this.name = name` bits we see here.

Look at the default/optional params in the API for guidance.

I am pretty sure that `id`, `created`, `status` should all be optional.  `memberIds` and `adminUserIds` can default to an empty set (even though they will fail API side).  We don't have a mechanism for throwing errors in this client so I am actually ok with that.